src/Ares.php

```diff
@@ -1,46 +1,94 @@
 <?php
+/**
+ * This file is part of Ares.
+ * Copyright (c) 2015 Václav Vrbka (http://aurielle.cz)

```

Comment:
Good point.

Probably the correct way is to have his name:
- in Ares\Driver\HttpGet
- in library README.md - because he started coding this library; so there he should be author of original implementation